frontend/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
@pcgamer1 I still see two .gitignores which can stay but they have repeated stuff, eg. node_modules. Make one additional  .gitignore on the parent level that is common to both so you can put node_modules there instead of duplicating what you want to ignore.